python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -260,23 +260,6 @@ def Index__new__(cls, *args, **kwargs):
     return self
 
 
-def name(self):
-    return self._fsproxy_wrapped._name
-
-
-def Index__setattr__(self, name, value):
```

Comment:
Just a note: This is the same `__setattr__` method from `_FastSlowProxy` except customized to set the `_name` attr(`_names` for MultiIndex).